2020-01-25 14:21:30.342 [main] ERROR g.proj.javao2o.service.impl.WechatAuthServiceImpl -insertWechatAuth error:org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dfrrrrrrrr' for key 'open_id'
### The error may exist in file [D:\ClassFiles\WebMarketplaceOnCampus\spring-boot-demo\target\classes\mapper\WechatAuthMapper.xml]
### The error may involve guo.proj.javao2o.dao.WechatAuthDao.insertWechatAuth-Inline
### The error occurred while setting parameters
### SQL: INSERT       INTO       tb_wechat_auth(user_id,open_id,create_time)       VALUES       (?,?,?)
### Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dfrrrrrrrr' for key 'open_id'
; Duplicate entry 'dfrrrrrrrr' for key 'open_id'; nested exception is java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dfrrrrrrrr' for key 'open_id'
2020-01-25 14:21:30.345 [main] ERROR g.proj.javao2o.service.impl.WechatAuthServiceImpl -insertWechatAuth error:org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dfrrrrrrrr' for key 'open_id'
### The error may exist in file [D:\ClassFiles\WebMarketplaceOnCampus\spring-boot-demo\target\classes\mapper\WechatAuthMapper.xml]
### The error may involve guo.proj.javao2o.dao.WechatAuthDao.insertWechatAuth-Inline
### The error occurred while setting parameters
### SQL: INSERT       INTO       tb_wechat_auth(user_id,open_id,create_time)       VALUES       (?,?,?)
### Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dfrrrrrrrr' for key 'open_id'
; Duplicate entry 'dfrrrrrrrr' for key 'open_id'; nested exception is java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dfrrrrrrrr' for key 'open_id'
2020-01-25 14:23:09.064 [main] ERROR g.proj.javao2o.service.impl.WechatAuthServiceImpl -insertWechatAuth error:org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dfrrrrrrrr' for key 'open_id'
### The error may exist in file [D:\ClassFiles\WebMarketplaceOnCampus\spring-boot-demo\target\classes\mapper\WechatAuthMapper.xml]
### The error may involve guo.proj.javao2o.dao.WechatAuthDao.insertWechatAuth-Inline
### The error occurred while setting parameters
### SQL: INSERT       INTO       tb_wechat_auth(user_id,open_id,create_time)       VALUES       (?,?,?)
### Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dfrrrrrrrr' for key 'open_id'
; Duplicate entry 'dfrrrrrrrr' for key 'open_id'; nested exception is java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dfrrrrrrrr' for key 'open_id'
2020-01-25 16:05:08.169 [main] ERROR g.proj.javao2o.service.impl.WechatAuthServiceImpl -insertWechatAuth error:org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dfrrr9rrrr' for key 'open_id'
### The error may exist in file [D:\ClassFiles\WebMarketplaceOnCampus\spring-boot-demo\target\classes\mapper\WechatAuthMapper.xml]
### The error may involve guo.proj.javao2o.dao.WechatAuthDao.insertWechatAuth-Inline
### The error occurred while setting parameters
### SQL: INSERT       INTO       tb_wechat_auth(user_id,open_id,create_time)       VALUES       (?,?,?)
### Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dfrrr9rrrr' for key 'open_id'
; Duplicate entry 'dfrrr9rrrr' for key 'open_id'; nested exception is java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dfrrr9rrrr' for key 'open_id'
